### ISSUE TITLE
JetEngine - the sorting option doesn't work with User listings

### ISSUE THREAD

**Issue by honlapdavid** [CrocoBlock/suggestions#2404]
Hello,

in the JetEngine the sorting option doesn't work with User listings. Can you add (fix this problem) in the future?

Thank you.

**Comment by mfrendi** [comment]
Also looking for this feature as well

**Comment by rentaterry** [comment]
having the same problem. Crocoblock replied there is no such function. Hope they can add it in future.

**Comment by kochhase** [comment]
yes please! For now you have to do your own code. But this is only working with standard user fields like ID or Date.
